resources/views/components/menu/account.blade.php

```diff
@@ -47,7 +47,7 @@
     </div>
     <x-nav-dropdown trigger-class="py-0" dropdown-class="dropdown-menu-right" :desktopHref="route('user.show', $user)">
         <x-slot name="trigger">
-            <x-user.avatar :user="$user" display="icon" :hasHref="false" iconSize="sm" :tooltip="false" class="rounded-sm" />
+            <x-user.avatar :user="$user" display="icon" :hasHref="false" iconSize="sm" :tooltip="false" class="rounded-sm userpic" />
```

Comment:
This fixes a bug where the settings page is trying to do some DOM manipulation to update the user's avatar after a new file upload. Currently the selector quietly fails because it's trying to select based on this classname, which is missing.